packages/builder/src/pages/builder/app/[application]/automation/_flagged/index.new.svelte

```diff
@@ -24,21 +27,38 @@
   import { sdk } from "@budibase/shared-core"
   import TopBar from "@/components/common/TopBar.svelte"
   import { BannerType } from "@/constants/banners"
-
-  enum Filter {
-    All = "All automations",
-    Published = "Published",
-    Drafts = "Drafts",
-  }
+  import { capitalise, durationFromNow } from "@/helpers"
 
   let showHighlight = true
   let createModal: ModalAPI
   let updateModal: Pick<ModalAPI, "show" | "hide">
   let confirmDeleteDialog: Pick<ModalAPI, "show" | "hide">
   let webhookModal: ModalAPI
-  let filter = Filter.All
+  let filter: PublishResourceState | undefined
   let selectedAutomation: Automation | undefined = undefined
 
+  const filters: {
+    label: string
+    filterValue: PublishResourceState | undefined
+  }[] = [
+    {
+      label: "All automations",
+      filterValue: undefined,
+    },
+    {
+      label: "Published",
+      filterValue: PublishResourceState.PUBLISHED,
+    },
+    {
+      label: "Disabled",
+      filterValue: PublishResourceState.DISABLED,
+    },
+    {
+      label: "Drafts",
```

Comment:
Should this be "Unpublished" too? Or should the labels on the automation list say "Drafts"?